Updated the README.md document to be a little more robust.

mathew-thomson opened the pull request:
Fleshed this out using other documentation we have laying around. May need some additional fine-tuning.

mathew-thomson committed:
Updated the README.md document to be a little more robust.

Fleshed this out using other documentation we have laying around. May need some additional fine-tuning.

DougManuel committed:
Addition of CoVaRR Net

DougManuel commented on README.md:
```diff
@@ -1,2 +1,20 @@
+To ensure that the COWS database is usable for all users, we have also developed a controlled vocabulary and a suite of related tools called the Open Data Model ([ODM](https://github.com/Big-Life-Lab/ODM)). The ODM an open, standard approach to share wastewater surveillance data. The ODM helps the wastewater surveillance community collaborate by allowing teams worldwide to share their data in a c ...
+
+## Current Objectives: 
+- **Create the most robust open data model for wastewater and environmental epidemiology and surveillance.** Supported by an international steering committee, the ODM dictionary will allow consistent reporting of SARS-CoV-2, including variants and mutations. The ODM will support reporting for other health hazards, including other microbes and chemicals. Reporting will expand beyond wastewater to  ...
```
I know it is confusing. I suggest that we move these objectives to the ODM repository, since they are for the ODM.

It is a good idea to have COWS objectives.
- Easily share WW data. We focus on Canadian data but we are happy to receive other data.
- Allow detailed WW data (this is a distinction between COWS and other repositories - we can make reference to World Sphere and Norman).
- Easy to add data. This is where we'd mention that we use ODM tools to help validate data. We can offer to help people add data. We can automatically add your data if your data ODM format on common data platforms like Dropbox, Sync, Google Drive, ArcGIS.

mathew-thomson replied:
Added these goals and a mention of the "scrapping from COWS-private" structure in the latest commit - let me know if I caught everything.

DougManuel reviewed (changes requested):
@wyusuf068 and @mathew-thomson I suggest that we have two repositories or COWS. `COWS-private` for where we scrap data. We have this a private site so that we don't expose the scrap code.
`COWS` in turn, we scrap `COWS-private` from `COWS`. We can be upfront about this process.

mathew-thomson committed:
Update goals, mention COWS-private

Amended the readme to update the goals and include mention of a private COWs repository that the public-facing COWS repository will scrap from.

wyusuf068 committed:
Merge branch 'main' into New-ReadMe-File-draft

wyusuf068 committed:
[Refactor] edit text
